build.gradle

```diff
@@ -122,6 +122,7 @@ allprojects {
         retrofitVersion = "2.6.2"
         staxApiVersion = "1.0-2"
         threeTenAbpVersion = "1.2.1"
+        threeTenBpVersion = "1.4.0"
```

Comment:
Why can't we use ThreeTenApb in the tests?

Reply:
Thanks - that makes sense. How hard would it be to just stick with java.util.Date & co? We could make some utility functions that accept ThreeTenABP instances and convert them to work with Date. I know that API sucks but if it's not a TON of work it might be nice to avoid adding this dep (even if it's only for test).

Reply:
I don't think it would be too hard but we have some classes like [DateTimeRfc1123](https://github.com/Azure/azure-sdk-for-android/blob/dev/sdk/core/azure-core/src/main/java/com/azure/android/core/util/DateTimeRfc1123.java) the where we still probably need to import this dependency for testing, which is what I'm doing right now on [this PR](https://github.com/Azure/azure-sdk-for-android/pull/214).